.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "Appium/tests/helpers/mocks"]
 	path = Appium/tests/helpers/mocks
 	url = https://github.com/jasonkarns/bats-mock
+[submodule "Appium/tests/helpers/bats"]
```

Comment:
Just did a quick test to make sure and unless travis has some custom logic, to update the submodule, one would need to do a pull in submodule directory and then make a commit in parent directory to save the updated state of submodule so it should not always use latest commit.